build.gradle

```diff
@@ -72,3 +76,17 @@ javaVersions.each { version ->
 tasks.withType(Test).configureEach {
     useJUnitPlatform()
 }
+
+gradlePlugin {
+    website = 'https://github.com/line/thrift-gradle-plugin'
+    vcsUrl = 'https://github.com/line/thrift-gradle-plugin'
+    plugins {
+        thriftGradlePlugin {
+            id = 'com.linecorp.thrift-gradle-plugin'
+            implementationClass = 'com.linecorp.thrift.plugin.ThriftPlugin'
+            displayName = 'A gradle plugin for compiling Thrift IDL files using thrift compiler'
```

Comment:
Maybe using `Thrift Gradle Plugin` for `displayName` is enough since it's `short displayable name for plugin` ?